### ISSUE TITLE
BSE bhavcopy with ISIN

### ISSUE THREAD

**Issue by tusharbhardwaj1989** [hemenkapadia/getbhavcopy#26]
Is there a method with which BSE bhavcopy can be downloaded with ISIN values?

**Comment by hemenkapadia** [comment]
Does BSE india provide ISIN values somewhere in the bhavcopy? Getbhavcopy downloads data directly from BSE, so unless it is available there, nothing much can be done.

**Comment by tusharbhardwaj1989** [comment]
Yes it does, if you check the second link. It provides the bhavcopy with isin values.

https://www.bseindia.com/markets/equity/EQReports/bhavcopy31012018.aspx?expandable=7

**Comment by tusharbhardwaj1989** [comment]
Hello Hemen,

you there?
